ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Windows_Updates/tasks/main.yml

```diff
@@ -4,13 +4,12 @@
 ###################
 - name: Download and Install Windows Updates
   win_updates:
-    register: update_result
+  register: update_result
   ignore_errors: yes
   tags: Windows_Updates
 
 - name: Reboot machine if necessary
   win_reboot:
-    shutdown_timeout: 3600
```

Comment:
If this was deprecated, what's replaced it?

Reply:
Nothing has directly replaced it. The shutdown_timeout option is the max amount of time to wait for shutdown to begin, and the reboot_timeout is the max amount of time to wait for machine to re-appear on the network and respond to a test command. It should work using only the reboot_timeout option.